shakespeare.cabal

```diff
@@ -12,7 +12,7 @@ description:
     .
     Note there is no dependency on haskell-src-extras. Instead Shakespeare believes logic should stay out of templates and has its own minimal Haskell parser.
     .
-    Packages that use this: shakespeare-js, shakespeare-css, shakespeare-text, hamlet, and xml-hamlet
+    Packages that use this: shakespeare-js, shakespeare-css, hamlet, and xml-hamlet
```

Comment:
shakespeare-js, shakespeare-css, and hamlet are all also deprecated in favor of shakespeare, can you remove those too?